Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM alpine:3.4
 
 RUN apk add --no-cache curl python
 
-ENV GOOGLE_CLOUD_SDK_VERSION=187.0.0
+ENV GOOGLE_CLOUD_SDK_VERSION=202.0.0
```

Comment:
The latest version is `207.0.0`, is there a reason not to update to the latest?

Reply:
not necessarily, would that be preferred?

Reply:
see https://github.com/NYTimes/drone-gke/pull/89